astropy/utils/data.py

```diff
@@ -160,9 +175,22 @@ def get_readable_fileobj(name_or_obj, encoding=None, cache=False,
         Timeout for remote requests in seconds (default is the configurable
         `astropy.utils.data.Conf.remote_timeout`, which is 3s by default)
 
+    sources : list of str, optional
+        If provided, a list of URLs to try to obtain the file from. The
+        result will be stored under the original URL. The original URL
+        will *not* be tried unless it is in this list; this is to prevent
+        long waits for a primary server that is known to be inaccessible
+        at the moment.
+
+    update_cache : bool, optional
+        If true, attempt to download the file anew; if this is successful
+        replace the current version. If not, raise a URLError but leave
+        the existing cached value intact.
+
     Returns
     -------
     file : readable file-like object
+
```

Comment:
Here and elsewhere: why the extra empty line at the end of the docstring? PEP 257 doesn't ask for it, and it just means that on one's screen one sees one less line of useful information if one does `...?`

Reply:
Confusion on my part. I think I misread some linter warning somewhere.